packages/flutter/lib/src/rendering/sliver_multi_box_adaptor.dart

```diff
@@ -517,13 +517,17 @@ abstract class RenderSliverMultiBoxAdaptor extends RenderSliver
   }
 
   @override
-  void debugFillDescription(List<String> description) {
-    super.debugFillDescription(description);
-    if (firstChild != null) {
-      description.add('currently live children: ${indexOf(firstChild)} to ${indexOf(lastChild)}');
-    } else {
-      description.add('no children current live');
-    }
+  void debugFillProperties(List<DiagnosticsNode> description) {
+    super.debugFillProperties(description);
+    description.add(
+      new ObjectFlagProperty<RenderBox>(
+        'firstChild',
```

Comment:
I agree that case was pushing use of ObjectFlagProperty too far.
The clear bridge too far was that the presentDescription was generated dynamically.
Switched to a couple message properties instead.
Done.